README.md

```diff
@@ -37,3 +40,14 @@ How to deploy the website to community-catalog.zooniverse.org:
   [Front-End-Monorepo](https://github.com/zooniverse/front-end-monorepo).
   Dependencies are kept at a minimum to make maintenance easier.
 - This repo should be archived after (TODO: INSERT INITIAL LIFESPAN DATE HERE).
+
+**Auth & Login**
+@shaunanoordin 2023.01.05
+- Unlike many of our earlier CFEs (custom front ends), the Community Catalog
+  doesn't use oAuth for authentication.
```

Comment:
It does use OAuth. That’s why you have to send a OAuth bearer token with API requests.

Reply:
👌 Good point, sentence has been corrected. Thanks!